packages/garbo/src/tasks/dailyItems.ts

```diff
@@ -138,6 +140,62 @@ function pickCargoPocket(): void {
   }
 }
 
+function bestDevilerCandy(): Item {
+  // These are replenishable notrade nodiscard items that we would prefer to use
+  const priorityUntradeableNoDiscardList =
+    $items`sugar shotgun, sugar shillelagh, sugar shank, sugar chapeau, sugar shorts, sugar shield, sugar shirt`.filter(
+      (i) => itemAmount(i) > 1,
+    );
+  const bestPriorityCandy =
+    priorityUntradeableNoDiscardList.length > 0
+      ? maxBy(priorityUntradeableNoDiscardList, (i) => itemAmount(i))
+      : null;
+
+  if (bestPriorityCandy) return bestPriorityCandy;
```

Comment:
```suggestion
  if (priorityUntradeableNoDiscardList.length > 0) {
    return maxBy(priorityUntradeableNoDiscardList, itemAmount);
  }
```

Reply:
I think this is a nicer pattern given we're already doing an early return, you know?